<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IpSettingsWizardPage</class>
 <widget class="QWizardPage" name="IpSettingsWizardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>444</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>510</width>
    <height>444</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>510</width>
    <height>444</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>On/Off Host Settings</string>
  </property>
  <attribute name="pageId">
   <string notr="true">1</string>
  </attribute>
  <widget class="QGroupBox" name="groupBox_newset">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>231</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Settings</string>
   </property>
   <widget class="QPushButton" name="pushButton_set">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>211</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>IP address:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_ip">
       <property name="whatsThis">
        <string notr="true"/>
       </property>
       <property name="inputMask">
        <string notr="true"/>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Subnet mask:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_sm">
       <property name="whatsThis">
        <string notr="true"/>
       </property>
       <property name="inputMask">
        <string notr="true"/>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Default gateway:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_dg">
       <property name="whatsThis">
        <string notr="true"/>
       </property>
       <property name="inputMask">
        <string notr="true"/>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_fill">
       <property name="text">
        <string>Fill Default Value</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_oldset">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>201</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Current Settings</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>181</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>IP address:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_oldip">
       <property name="whatsThis">
        <string notr="true"/>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(240, 240, 240);</string>
       </property>
       <property name="inputMask">
        <string notr="true"/>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Subnet mask:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_oldsm">
       <property name="whatsThis">
        <string notr="true"/>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(240, 240, 240);</string>
       </property>
       <property name="inputMask">
        <string notr="true"/>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Mac address:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_mac">
       <property name="whatsThis">
        <string notr="true"/>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(240, 240, 240);</string>
       </property>
       <property name="inputMask">
        <string notr="true"/>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>69</y>
     <width>441</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Network Interfaces</string>
   </property>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_lock">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>29</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
